spec/services/identity_linker_spec.rb

```diff
@@ -26,6 +26,20 @@
       expect(identity_attributes).to include new_attributes
     end
 
+    it 'does not write last_authenticated_at when :include_identity_session_attributes is false' do
+      identity = IdentityLinker.new(user, service_provider).
+        link_identity(include_identity_session_attributes: false)
+
+      aggregate_failures do
+        expect(identity.last_authenticated_at).to be_nil
+        expect(identity.session_uuid).to be_nil
+        expect(identity.access_token).to be_nil
+        expect(identity.last_ial1_authenticated_at).to be_nil
+        expect(identity.last_ial2_authenticated_at).to be_nil
+        expect(identity.verified_at).to be_nil
```

Comment:
Just because my brain has been so focused on `last_consented_at`&mdash;this whole change is unrelated to that, and is looking ahead to the next step, right?

Reply:
yeah I just wanted to make sure that the row was as blank as possible